src/Agent/NewRelic/Agent/Core/Agent.cs

```diff
@@ -477,6 +477,23 @@ public void RecordLlmEvent(string eventType, IDictionary<string, object> attribu
             _customEventTransformer.Transform(eventType, attributes, transaction.Priority);
         }
 
+        public IEnumerable<string> GetConfiguredDTHeaders()
```

Comment:
We should probably return either List<string> or IList<string> here so that when we evaluate the count, we don't have to enumerate the IEnumerable.